README.md

```diff
@@ -9,9 +9,25 @@ The Pet Rescue app is derived from the [Baja Pet Rescue Dog Adoption Application
 
 # 🚀 Getting Started
 
-Let's get your machine setup to startup the application!
+Let's get your machine setup to start up the application!
 
-## Prerequisites
+## Docker
+
+Install a containerization app, such as Docker Desktop.
+
+Clone the repo to your local machine and navigate to the directory and run:
+
+`docker-compose build` to build the base image.
```

Comment:
these days, the command is `docker compose`  (no dash).  It's a newer version of the subcommand